common/v2/config/tokens/kovan.json

```diff
@@ -15,13 +15,13 @@
   },
   {
     "address": "0x4c38cdc08f1260f5c4b21685654393bb1e66a858",
-    "symbol": "DAI",
+    "symbol": "KovanDAI",
```

Comment:
@FrederikBolding  @blurpesec  Just want to double check about this.
From what I gather, we change the token symbol to distinguish between each token when we display the totals in Walletbreakdown.
Taking this path would mean we need to create a unique symbol.
And that the conflict will still arise for custom tokens etc.
And we can't get balances for tokens on other networks (not sure this is even possible today).
Shouldn't we modify the `totals` function instead and take the network into account as well ?

Reply:
Currently we rely too much on the symbols being unique, this is indeed a temporary solution. 

We assume they are unique when looking up rates, giving wrong rates for total balances, but we also assume they are unique when initializing cache here: https://github.com/MyCryptoHQ/MyCrypto/blob/281717a62c8ae71561242b184c65f1c870da9225/common/v2/config/cacheData.ts#L60

If you want, a more permanent solution to at least one of the problems in changing how we initialize the assets into cache?

Reply:
As discussed with @blurpesec on Slack - having unique symbols for testnet tokens makes it much less confusing for users as well.